vg/draw/canvas.go

```diff
@@ -32,8 +32,47 @@ type TextStyle struct {
 
 	// Font is the font description.
 	Font vg.Font
+
+	// Rotation is the text rotation in radians, counterclockwise from
```

Comment:
Does it need to be said? The directions handled are left or right and a quarter turn rotation ccw results in the same relative transform for both.